nvflare/fuel/f3/cellnet/cell.py

```diff
@@ -226,11 +232,15 @@ def _fire_and_forget(
             targets = [targets]
 
         result = {}
+        futures = {}
         for target in targets:
-            self.send_blob(
+            future = self.send_blob(
                 channel=channel, topic=topic, target=target, message=message, secure=secure, optional=optional
             )
+            futures[target] = future
             result[target] = ""
+        message.set_prop(MessagePropKey.FUTURES, futures)
```

Comment:
Are we using this MessagePropKey.FUTURES anywhere?

Reply:
In cell.py.

Reply:
I search and can't find where we get this prop?

Reply:
It's used in the file_sender.py which is removed. But the prop should stay for future use.